packages/spright-components/src/all-components.ts

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
At some point we'll need to figure out how the all-components approach will work if someone wants all of spright and all of nimble. Can I import both all-components files or will that result in errors from nimble elements being defined twice?

That doesn't necessarily need to be part of this PR.

Reply:
I assume we're going to need an `all-components.js` in Spright for the sake of `SprightBlazor`, otherwise, I doubt we need to support that use case for clients. It's not likely we'll have tons of Spright components, nor that any given client app would want to import all of them.

Reply:
Yeah, we can work it out when we get to SprightBlazor.

Reply:
I think we should go ahead and import the nimble all-components from the spright all-components. The all-components biggest user is Blazor but it's not the only user [(codesearch)](https://codesearch.natinst.com/search/text?q=all-components-bundle%20-repo%3Animble%20-path%3A.cshtml&fold_case=auto&regex=true&context=true) [(GitHub)](https://github.com/search?q=org%3Ani+all-components-bundle+-repo%3Ani%2Fnimble+-repo%3Ani%2Fspright&type=code).

As-is I think a user that tries to use both nimble all-components-bundle and spright all-components-bundle in a page from separate builds could have issues because each all-components-bundle would load their own copy of FAST element, etc which is at best extra bulk and at worst (and I think expected) will cause conflicts and issues.

Reply:
Also see the statement from the other comment about being derived from FAST Foundation but I think we just keep it even simpler and derive from FASTElement. I kinda expect long-term most things in FAST Foundation to be in Nimble instead of spright and even if extending from FAST Foundation nimble has plenty of examples already.

My understanding is nimble-rectangle is super simple and something just for infrastructure set-up that we can remove when we get a real spright component